index.html

```diff
@@ -14,15 +14,16 @@
 
     <link rel="stylesheet" href="bootstrap.min.css" integrity="sha384-2hfp1SzUoho7/TsGGGDaFdsuuDL0LX2hnUp6VkX3CUQ2K4K+xjboZdsXyp4oUHZj" crossorigin="anonymous">
 
-    <link rel="stylesheet" href="style.css" />
-
+    <link rel="stylesheet" href="style.css">
+    <link rel="icon" href="gfx/favicon.svg">
   </head>
 
   <body>
     <section class="jumbotron text-xs-center">
       <div class="container">
-        <h1 class="jumbotron-heading display-1">Iris Project</h1>
-        <p class="lead">Iris is a Higher-Order Concurrent Separation Logic Framework, <br> implemented and verified in the Coq proof assistant.</p>
+        <img src="gfx/logo.svg" style="height:140px; margin-bottom:20px;" alt="Iris logo">
+        <!--<h1 class="jumbotron-heading display-1">Iris Project</h1>-->
+        <p class="lead">A Higher-Order Concurrent Separation Logic Framework, <br> implemented and verified in the Coq proof assistant.</p>
```

Comment:
This is not a full sentence any more, so I am not sure if having a `.` at the end makes sense?

Reply:
I don't know, I think I'd keep it. But let's see if there are other opinions.

Reply:
I agree with Ralf, the period does not make sense.